indico/modules/rb_new/client/js/modules/calendar/reducers.js

```diff
@@ -24,6 +24,7 @@ import {requestReducer} from 'indico/utils/redux';
 import {actions as bookRoomActions} from '../../modules/bookRoom';
 import * as actions from '../../actions';
 import * as calendarActions from './actions';
+import * as bookingActions from '../../common/bookings/actions';
```

Comment:
import {actions as bookingActions} from '../../common/bookings';

Reply:
Then why do we have sth similar above?

Reply:
`calendarActions` is from the same module